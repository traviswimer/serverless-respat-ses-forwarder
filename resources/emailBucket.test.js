const emailBucket = require('./emailBucket');

describe('emailBucket()', () => {
	test('generates bucket', () => {
		expect(emailBucket({
			prefix: "ThePrefix",
			pattern_name: "ThePatternName",
			bucket_name: "TheBucketName"
		})).toMatchSnapshot();
	});
});
