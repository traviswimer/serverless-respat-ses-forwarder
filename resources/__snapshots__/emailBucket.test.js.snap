// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`emailBucket() generates bucket 1`] = `
Object {
  "Properties": Object {
    "AccessControl": "Private",
    "BucketName": "TheBucketName",
  },
  "Type": "AWS::S3::Bucket",
}
`;
